raqm.c

```diff
@@ -573,7 +581,6 @@ raqm_shape_u32 (unsigned int* u32_str,
         }
         else
         {
-            int j;
```

Comment:
The change was to make the declarations `raqm.c:652` and `raqm.c:576` one declaration.